.appveyor.yml

```diff
@@ -13,6 +13,10 @@ environment:
     secure: 96fJ3r2i2GohbXHwnSs5N4EplQ7q8YmLpPWM0AC+f4s=
   CODECOV_TOKEN:
     secure: XN4jRtmGE5Bqg8pPZkwNs7kn3UEI73Rkldqc0MGsQISZBm5TNJZOPofDMc1QnUsf
+  OAUTH2_TOKEN_FILE_KEY:
```

Comment:
@ei-grad @efiop should we keep these secrets out of git? can I get them to decrypt the keys and write/delete on GDrive?

Reply:
the same true for AWS and other clouds btw, it looks very dangerous that I can use them easily to do anything I want in the cloud

Reply:
@shcheklein What do you mean? These secrets are encrypted already.

Reply:
@efiop do you meant that _all_ these secrets (including AWS, GCP, etc) are encrypted by some key that is specific to appveyor?

Reply:
Yes, to travis and appveyor. And that key is only available during non-PR builds, so it is harder to steal them.

Reply:
👍